File: 06 telefonski_stub/telefonski_stub.cpp
#include <iomanip>
#include <iostream>

using namespace std;

int main(int argc, char const *argv[]) {
  double h, a;
  cin >> h >> a;
  // b predstavlja visinu na kojoj se stub prelomio
  double b = 0.5 * (h - (a * a) / h);
  cout << fixed << setprecision(3) << b << endl;
  return 0;
}

File: 06 telefonski_stub/telefonski_stub.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace telefonski_stub
{
    class Program
    {
        static void Main(string[] args)
        {
            double h = double.Parse(Console.ReadLine());
            double a = double.Parse(Console.ReadLine());
            // b predstavlja visinu na kojoj se stub prelomio
            double b = 0.5 * (h - (a * a) / h);
            Console.WriteLine(b.ToString("0.000"));
        }
    }
}
